l5kit/l5kit/evaluation/metrics.py

```diff
@@ -1,7 +1,10 @@
-from typing import Callable
+from enum import IntEnum
+from typing import Callable, Optional, Tuple
 
 import numpy as np
 
+from l5kit.planning import utils
```

Comment:
at this point I would just move the stuff we need here and delete `l5kit.planning.utils`. My suggestion was more about not replicating code (and fix the notebook) than to keep that file. Sorry I should have clarified that from the start :)

Reply:
No problem, let's merge this one, and then later I can move things around, there are things we should probably move to a "geometry" package or something similar, but we can discuss that later.